components/gitpod-protocol/src/messaging/error.ts

```diff
@@ -112,7 +112,7 @@ export const ErrorCodes = {
     SNAPSHOT_ERROR: 630 as const,
 
     // 640 Headless logs are not available (yet)
-    HEADLESS_LOG_NOT_YET_AVAILABLE: 640 as const,
+    HEADLESS_LOG_NOT_YET_AVAILABLE: 444 as const,
```

Comment:
I checked the places where we could filter this from the top of my head:
 - https://github.com/gitpod-io/gitpod/blob/075e14c47f3cadf0e4205fa9a954daf5427b3f70/components/server/src/websocket/websocket-connection-manager.ts#L378-L419 :heavy_check_mark: 
 - https://github.com/gitpod-io/gitpod/blob/b766c9146ebd90da00547eeae718d371c533fc2c/components/gitpod-protocol/src/messaging/error.ts#L30-L34 :heavy_check_mark: 

Anything else we can think of? :thinking: